Slingshot.Api/Concrete/BitbucketRepository.cs

```diff
@@ -90,6 +90,14 @@ public BitbucketRepository(Uri uri, string host, string token)
                     _branch = queryStrings["at"];
                     return _branch;
                 }
+
+                if (queryStrings["pr"] != null)
+                {
+                    string pullRequestId = queryStrings["pr"];
+                    var pr = await this.GetPullRequest<BitBucketPullRequest>(pullRequestId);
```

Comment:
if change to use IPullRequestInfo, we will get failed to instantiate IPullRequestInfo. 
i think we have to use a concrete class inside "<>". but the "var" can be replaced by "IPullRequestInfo"